Program.cs

```diff
@@ -1,10 +1,6 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿//Print multiplication tables from 1 to 10
+
+for (int i = 1; i <= 10; i++) {
+   for (int j = 1; j <= 10; j++) Console.WriteLine ($"{i} * {j} = {i * j}");
+   Console.WriteLine ();

```

Comment:
I an getting this output when I am running your program more than one time. Can you please check this and fix if this is reproducible in your computer.
![image](https://github.com/RaaneshKumar/Training/assets/94427955/9e930387-3531-4db4-83f7-2e05f6922c02)

Reply:
This is not reproducible in my system.